src/include/network/remote/dialogs.rb

```diff
@@ -142,12 +149,15 @@ def RemoteMainDialog
       if ret == :next
         CWMFirewallInterfaces.OpenFirewallStore(firewall_widget, "", event)
 
-        allowed = Convert.to_boolean(UI.QueryWidget(Id(:allow), :Value))
+        allowed_xinetd = Convert.to_boolean(UI.QueryWidget(Id(:allow_xinetd), :Value))
+        allowed_vncmanager = Convert.to_boolean(UI.QueryWidget(Id(:allow_vncmanager), :Value))
```

Comment:
we try to avoid using helpers which were introduced during ycp -> ruby conversion in the new code. You can use just ```UI.QueryWidget(...)``` here